src/test/java/com/wikia/webdriver/common/core/Assertion.java

```diff
@@ -77,6 +77,12 @@ public static void assertEquals(String current, String pattern) {
     );
   }
 
+  public static void assertEqualsIgnoreCase(String current, String pattern) {
+    current = current.toLowerCase();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Introduce a new variable instead of reusing the parameter "current". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1226)